index.d.ts

```diff
@@ -1159,6 +1163,12 @@ declare namespace Eris {
     reason?: string;
     user: User;
   }
+  interface GuildIncidentData {
+    dmsDisabledUntil: number | null;
+    dmSpamDetectedAt: number | null;
```

Comment:
gonna nitpick here dmSpam should technically go before dmsDisabled

Reply:
Is the alphabet down south different to the one up here? Up North, D usually comes before P. Or does uppercase take priority for some reason

Reply:
oh no i completely misread that oops. also sheep.